messages/run-dfa.js

```diff
@@ -47,8 +47,8 @@ Use --rule-thread-timeout to increase or decrease the maximum runtime for a sing
 Use --sfgejvmargs to pass Java Virtual Machine args to override system defaults while executing Salesforce Graph Engine's rules.
 The example overrides the system's default heap space allocation to 8 GB and decreases chances of encountering OutOfMemory error.
 	$ sfdx scanner:run:dfa --sfgejvmargs "-Xmx8g" ...
-Use --with-pilot to allow execution of pilot rules:
-This example allows pilot rules in the "Performance" category to execute.
+Use --with-pilot to allow execution of experimental rules.
```

Comment:
@jfeingold35 I'm confused what this PR is about? This looks like a doc change, not updating the UI text only? Confused.
Either way, per the UI text doc Approved column, we're not using 'experimental' but 'pilot.'

https://docs.google.com/document/d/1oMhNKy1RbwLBs_ScvIRp8m6jlNT9WPlvWi1ew17U_JM/edit?usp=sharing